README.md

```diff
@@ -1,24 +1,246 @@
-# README
+# BillsBuddy
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div id="top"></div>
+<!-- PROJECT LOGO -->
+<div align="center">
+  <a href="https://github.com/fobadara/billsbuddy">
+    <img src="./app/assets/images/coin-logo.png" alt="Logo" width="80" height="80">
+  </a>
 
-Things you may want to cover:
+  <h1 align="center">BillsBuddy</h1>
 
-* Ruby version
+  <p align="center">
+    <br />
+    <a href="https://github.com/fobadara/billsbuddy#readme"><strong>Explore the docs »</strong></a>
+    <br /> 
+    <br />
+    <a href="https://drive.google.com/file/d/1cb_558Y_uxh1vsqGh0YQumVN5pZKZgLZ/view?usp=sharing">View Demo</a>
```

Comment:
[OPTIONAL] - KIndly change the title of this link to show it is for the video presentation